test/e2e/create_test.go

```diff
@@ -23,16 +21,16 @@ const (
 )
 
 func creationTestSuite(t *testing.T) {
-	numOfNodesToBeCreated := 1
-	t.Run("Creation", func(t *testing.T) { testWindowsNodeCreation(t, numOfNodesToBeCreated) })
-	t.Run("ConfigMap validation", func(t *testing.T) { testConfigMapValidation(t, numOfNodesToBeCreated) })
-	t.Run("Secrets validation", func(t *testing.T) { testValidateSecrets(t, numOfNodesToBeCreated) })
+	nodeCount := 1
```

Comment:
Why are you renaming this? The new name `nodeCount` does not tell us that this is the number of nodes to be created. It implies this is the number of nodes at this instant which is not true.